robottelo/cleanup.py

```diff
@@ -34,6 +34,21 @@ def host_cleanup(host_id=None):
     entities.Host(id=host_id).delete()
 
 
+def repo_cleanup(repo_id=None):
```

Comment:
I think this change is appropriate within the scope of this PR, but it makes me think we can create a simple cleanup method that takes an entity type and ID to implement this cleanup pattern across every entity.

Reply:
Removed cleanup code for now.